### PR TITLE
sci-misc/boinc: Version bump and init script fix

### PR DESCRIPTION
**Pull request by Yamakuzure**
This pull request fixes Gentoo Bug #626958
  "sci-misc/boinc-7.6.33-r5 typos in /etc/init.d/boinc"

In five places the eerror function was spelled "eeror", which is
obviously wrong. This commit fixes that.

See: https://bugs.gentoo.org/show_bug.cgi?id=626958

In the meantime two new versions where release:
 - client_release/7.2/7.2.47
 - client_release/7.8/7.8.1

As a revision bump is needed to enforce the installation of the fixed
init script anyway, I took the liberty to bump those two versions in
one go.

Package-Manager: Portage-2.3.7, Repoman-2.3.3

### CODE REVIEW

**Comment by Yamakuzure** [comment]
@SoapGentoo @jlec : Sorry for not pinging you earlier. I wanted to try something out, and then ping you when I can write down the result. (Follows below)
However, family matters got in between... :-/

Unfortunately I can not build any series 7.2 client with gcc-6.4.0 any more. Patching that isn't too hard, but I wanted to leave the fix out of this update, as fixing the init script looked to be a bit more important.

Speaking of importance, I am not convinced that we should really keep now 4 lines of boinc client software lounging around.

The current situation is:

* Berkeley insists on 7.2.42 as the default version for Linux and Mac OS X (10.4)
* For Windows, the 7.2 line isn't even mentioned any more.
* For Mac OS X (10.6+) and Windows the 7.6 series is recommended as the default version.
* For Linux the 7.4 series is called "development version", although there hasn't been any development for over three years, now.

What I would like to do, now that there is the new series 7.8 as a new development version, is to ditch both 7.2 and 7.4.

However, there is a catch:
The 7.2 series is the last, that does not need the USE="webkit" enabled x11-libs/wxGTK.
And the series just got an update. So it is still actively maintained.

...mayby just ditch the 7.4 series? I think this one, at least, is superfluous.

**Comment by Yamakuzure** [comment]
@mgorny I hope I got it this time. Maybe I should re-read your new proxy maintainer wiki page... ;-)

File: sci-misc/boinc/files/boinc.init.in [hunk 1]
```diff
@@ -15,13 +15,13 @@ create_work_directory() {
 	if [ ! -d "${RUNTIMEDIR}" ]; then
```
Comment (mgorny) [inline sci-misc/boinc/files/boinc.init.in]: But you're not bumping the version in this commit which makes the commit message wrong and confusing. Focus summary line on telling what you're fixed exactly, you've got enough space there to put shortly what exact issue is fixed. 

Also, the footer tag is `Bug:`, not `See:`. Please also remove the empty line between it and `Package-Manager`.

File: sci-misc/boinc/Manifest [hunk 2]
```diff
@@ -1,4 +1,5 @@
-DIST boinc-7.2.44.tar.gz 46626134 SHA256 bae214f238b8b3d1826a44e3a9e5d91cbad165e7df6daf8a7f208f36c51c5425 SHA512 60f58eecbe1f8d1a272b338d0c2b2fcbfb95538c448cbfdab25d3e860129b519a2c3b2608a909e285bdf4378c23a71107a6d3c2d3aa519ec906480d81be19ca9 WHIRLPOOL f071fce088769ca35bacc9466e3a863bc619d025e14aa26b96a5a736027e4438a2d660df804d95cc635c4bd83e636c0c79e2712f2953bbc2fd905091ec1a1589
```
Comment (mgorny) [inline sci-misc/boinc/Manifest]: This doesn't belong to this commit.

**Comment by Yamakuzure** [comment]
@mgorny Okay. My new try to separate it into logical changes are:
1) The fix, with revision bump (force fixed script over broken one)
2) Version bump of the series 7.2 client
3) Introduction of the 7.8 series client.

**Comment by mgorny** [comment]
Thanks a lot, merging now.

**Comment by mgorny** [comment]
It seems I failed to add the `Closes` line. This was merged as 42e99588ad3b1c7822cf9f150d4af33eb9c73517...192b9a6ad983e4419319fd2792ed344d73a754e1
